odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: Jupyterhub-dashboard
```

Comment:
hrmm I don't think we need a `-dashboard` suffix, since it's already of `kind: GrafanaDashboard`

Reply:
fixed :+1: